package.json

```diff
@@ -84,6 +86,7 @@
     "node-sass": "4.12.0",
     "po2json": "1.0.0-alpha",
     "postcss-loader": "3.0.0",
+    "prettier": "git+https://github.com/Automattic/wp-prettier.git#wp-prettier-1.17.0",
```

Comment:
We should not be using calypso standards here.  Our plugin is distributed in wp.org so should follow WP/GB javascript standards.

An example where things deviate is with arrow functions:

WP/GB:

```js
const someCallback = ( args ) => value
```

This config:
```js
const someCallback = args => value
```

Reply:
This should be changed as discussed in Slack. :+1: